### DISCUSSION TITLE
How to restart recording after turning off from heat

### DISCUSSION THREAD

**Discussion by Necron1** [gopro/labs#906] [Q&A]
I've got a HERO11 and my mate has a HERO12 that we're trying to record some of our homemade solar projects but the recording keeps stopping due to the cameras getting too hot. 

I'm trying to get them to automatically start recording once they've cooled down to not trigger the thermal overprotection with a boot script that checks if they've got USB power even when they've cooled down to a normal temperature. 

Command I used is: _!MBOOT="!Lbt"!SAVEbt=!MWAKE=2!MTUSB=1!>u0!S_
Firmware version is: _2.12.70_

![qrcode](https://github.com/gopro/labs/assets/37396542/4a25ce25-0280-4f4c-905a-a7e85b085baf)

**Comment by dnewman-gpsw** [comment]
Don't use permanent commands in boot scripts.

**Reply by Necron1** [reply]
No worries. So would I be right in flashing two QR codes, one with `!MTUSB=1!MWAKE=2!MQRDR=1` and then one with `!MBOOT="!Lbt"!SAVEbt=>u0!S`?

**Reply by dnewman-gpsw** [reply]
Yes.  Or one code like this `*TUSB=1*WAKE=2*QRDR=1*BOOT="!Lbt"!SAVEbt=>u0!S`   

`*` can be used instead of `!M` in the latest firmware.
